source/scripts/app.js

```diff
@@ -1014,6 +1031,19 @@ function addRecipe() {
   });
 }
 
+async function initializeDefaultCookbook() {
+  "use strict";
+
+  try {
+    await indexedDb.createCookbook("My cookbook", "Your default cookbook!");
+    await populateCookbooksPage().then(() => {});
```

Comment:
What is the purpose of the `then` clause?

Reply:
The then catches a the promise from populateCookbooksPage. If it's not there, an error gets reported in the console whenever populatCookbooksPage resolves false

Reply:
Got it